lib/measures/tbd/resources/psi.rb

```diff
@@ -1458,11 +1459,13 @@ def processTBD(os_model, psi_set, ioP = nil, schemaP = nil, g_kiva = false)
     dad   = s.surface.get.nameString
     id    = s.nameString
 
+    gross, pts = opening(os_model, id)
+
     # Site-specific (or absolute, or true) surface normal.
     t, r = transforms(os_model, space)
     n = trueNormal(s, r)
 
-    gross, points = opening(os_model, id, t)
+    points = (t * pts).map{ |v| Topolys::Point3D.new(v.x, v.y, v.z) }
```

Comment:
Topolys subsurface points are only populated (post-transformation) once new _rough opening_ vertices have been pre-validated by the _opening()_ function (entirely an OpenStudio-based operation).